### PR TITLE
slovak translation

### PR DESCRIPTION
**Pull request by sonics007**

### CODE REVIEW

**Comment by scakemyer** [comment]
Nice, but Last-Translator is missing a closing quote

**Comment by sonics007** [comment]
Sorry for that. its fixed now

File: resources/language/slovak/strings.po [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+msgid "General"
+msgstr "Všeobecné"
+
+msgctxt "#30001"
+msgid "Download path (not FAT32)"
+msgstr "Adresa pre stiahnute súbory (not FAT32)"
+
+msgctxt "#30002"
+msgid "Keep files after playback/download stops"
+msgstr "Ponecha súbor po prehratí/stiahnutí
```
Comment (scakemyer) [inline resources/language/slovak/strings.po]: missing a closing quote here also

**Comment by scakemyer** [comment]
Merged in d0a0e63

**Pull request closed without merging**